itou/cities/management/commands/sync_cities.py

```diff
@@ -0,0 +1,115 @@
+import json
+
+import httpx
+from django.conf import settings
+from django.contrib.gis.geos import GEOSGeometry
+from django.core.management.base import BaseCommand
+from django.db import transaction
+from django.template.defaultfilters import slugify
+
+from itou.cities.models import City
+from itou.utils.sync import DiffItemKind, yield_sync_diff
+
+
+def fetch_cities(districts_only=False):
+    base_geo_url = (
+        f"{settings.API_GEO_BASE_URL}/communes?"
+        "fields=nom,code,codesPostaux,codeDepartement,codeRegion,centre&format=json"
+    )
+    if districts_only:
+        base_geo_url = f"{base_geo_url}&type=arrondissement-municipal"
+    return json.loads(httpx.get(base_geo_url).content.decode("utf-8"))
```

Comment:
```suggestion
    return httpx.get(base_geo_url).json()
```

Il serait bon de faire un `raise_for_status` pour avoir un rapport d’erreur correct si la réponse n’est pas 200, plutôt qu’un problème pour décoder du JSON depuis le contenu d’une page d’erreur (sûrement HTML)